spec/models/patient_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe Patient, type: :model do
+  before(:all) do
+    @patient1 = FactoryBot.create(:patient)
+  end
+
+  describe 'valid patient factory' do
+    context 'factory' do
+      it 'is valid with valid attributes' do
+        expect(@patient1).to be_valid
+      end
+    end
+  end
+
+  it 'is not valid without a full name' do
+    patient2 = FactoryBot.build(:patient, full_name: nil)
```

Comment:
you can follow this tutorial https://semaphoreci.com/community/tutorials/working-effectively-with-data-factories-using-factorygirl
and configure FactoryBot so it allows you to write `build :patient` instead of `FactoryBot.build(:patient)`

Reply:
Fixed